marquez_client/file_backend.py

```diff
@@ -39,7 +41,9 @@ def put(self, path, headers, payload):
         put_details['headers'] = headers
         put_details['payload'] = payload
 
-        put_json = json.dumps(put_details)
+        put_json = jsonpickle.encode(
+            value=put_details,
+            fail_safe=Utils.handle_fail)
         log.info(put_json)
```

Comment:
If the json encoding fails, do we still want to log  the value of `put_json`?

Reply:
yes.